assets/javascript/game.js

```diff
@@ -0,0 +1,68 @@
+
+
+
+
+$(document).ready(function () {
+
+ // Define Functions
+
+  function getRandomInt(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+
+  function start_game() {
+    var totalscore =0;
+    wins=0;
+    losses=0;
+    var randomNum = getRandomInt(19,120); // Get a random number from 19-120 to compare total score with
+    console.log(randomNum);
+      
+    $("#score").text(randomNum); //store random number for current game's score
+
+    for (var i = 0; i < 4; i++) {
+      var crystals[i] = getRandomInt(1,9); //get 4 random #s for crystals 1-4
+
```

Comment:
Make sure you have the console open for chrome during development. I see that you have a syntax error somewhere around this line so always check console for errors.

Reply:
I updated this game over the break. Everything working. :)